src/test/scala/org/constellation/cluster/E2ETest.scala

```diff
@@ -150,155 +111,155 @@ class E2ETest
 
   }
 
+  "ConstellationApp" should "register a deployed state channel" in {
+    val deployResp = n1App.deploy("schemaString", "channelId")
+    deployResp.foreach { resp: Option[Channel] =>
+      sim.logger.info("deploy response:" + resp.toString)
```

Comment:
Wanted to check where and when those response messages would pop up. But at my local try to run the tests I get isses in E2E. Not sure if that's of any help, but it looks like 
```haskell
[IJ]sbt:constellation> testOnly *E2E*
2019-02-19 18:26:17,260 [pool-39-thread-1] INFO  o.c.crypto.KeyUtils$ - Insert provider return -1
[info] E2ETest:
[info] org.constellation.cluster.E2ETest *** ABORTED ***
[info]   java.lang.ExceptionInInitializerError:
[info]   at org.constellation.util.TestNode$.apply$default$2(TestNode.scala:17)
[info]   at org.constellation.E2E.createNode(E2E.scala:49)
```

When I just run `test` the first that complains is `SingleNodeGenesisTest` also failing in the TestNode.scala:17 line.

---
Ignore this if I'm missing something here (like a setup that those tests need or something). Might also be an issue quite beyond this branch.